docs/interpreter/spark.md

```diff
@@ -332,6 +337,19 @@ utilizing Zeppelin's built-in [Angular Display System](../usage/display_system/a
 
 <img class="img-responsive" src="{{BASE_PATH}}/assets/themes/zeppelin/img/docs-img/matplotlibAngularExample.gif" />
 
+## Running spark sql concurrently
+By default, each sql statement would run sequentially in `%spark.sql`. But you can run them concurrently by following setup.
+
+1. set `zeppelin.spark.concurrentSQL` to true to enable the sql concurrent feature. And also set `zeppelin.spark.concurrentSQL.max` to control the max number of sql statements running concurrently.
```

Comment:
should add that by setting this it would also change to the fair scheduler?

Reply:
Actually this is a bug of existing zeppelin. `zeppelin.spark.concurrentSQL` won't make spark run under fair scheduler.  What I did in this PR is that if user set `zeppelin.spark.concurrentSQL` to be true, then I will set spark conf `spark.scheduler.mode` to `FAIR` https://github.com/apache/zeppelin/pull/3044/files#diff-421cf195e8f8d49c7ca358768c9f56ceR96

Reply:
Sorry, I misunderstand your question. Will add that